src/briefcase/console.py

```diff
@@ -719,6 +719,23 @@ def input(self, prompt: str, *, markup: bool = False):
 
         return input_value
 
+    def parse_boolean(value: str) -> str:
```

Comment:
Return type declaration should be bool; and if it's a member function, the first argument should be `self`:
```suggestion
    def parse_boolean(self, value: str) -> bool:
```

However - there's an argument to be made that it shouldn't be a member function of the Console class, but a general utility functionin `briefcase.utils` - after all, there's nothing "console" specific about the parsing that is being done.

Reply:
That was a silly oversight on my part with the return value, Was thinking that there would be a better place for this function as well since its a more utility method but wasn't sure to put it, I had a look and couldn't see the briefcase utils except for some files present in the tests directory and one in the macOS directory, is this a separate repo you're referring to or am I just missing it ?

Reply:
My apologies - the reference to a `utils` module was a red herring. I forgot there's only a `utils` module in the `tests` folder.

If we were going to move it to a different module, it would likely be the `config` module on the basis that it's parsing a configuration value... but that's a bit of a stretch. 

However, I think it would be worth pulling it out as a standalone method, rather than being a member of Console. as noted previously, there's nothing "Console" specific about the method; plus, as currently defined, it's part of the public API of the Console class (because it's a non-underscore method). While it's a useful and valuable method, it's not something that needs to be part of Briefcase's public API.

Reply:
I've moved it into the config module as a solution for now, do we think their is a need to create a utils modules is there other more general utility functions present that would benefit from this ?

Reply:
We've survived so far without a utils module; I don't think this is enough of a trigger to add one now. 

The other option would be to include it in the `console` module itself - that would avoid the need for the import, but it's much of a muchness which one we actually use.